dev-java/batik/files/batik-squiggle-1.16.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Squiggle
+Comment=SVG browser
+Exec=batik-squiggle-1.16
+Icon=init
+Terminal=false
+Type=Application
+Categories=Graphics;VectorGraphics;
+MimeType=image/svg+xml;
```

Comment:
do we need a new file for each bump or could we just tag the existing file and replace the version using `sed` from the ebuild?

Reply:
> do we need a new file for each bump or could we just tag the existing file and replace the version using `sed` from the ebuild?

How should that work if more than one slot is installed?

Reply:
instead of version you put something like `%VER%` in the desktop file in `files` dir, you can also rename it to `*.template` if you want, and during installation you copy the file, `sed` the version in the copied file, and then you install the modified file using `newmenu`. you can see `thunderbird` ebuild for an example.

Reply:
I guess I'd prefer having it as is, needing a new file for each slot, simple and stupid. But I don't have a string opinion here.  Please feel free to improve as you like.
And please could you start the security stablereq when ready? Found that bug only this morning ...